Sources/TensorFlow/Optimizers/SGD.swift

```diff
@@ -25,7 +25,7 @@ public class SGD<Model: Differentiable>: Optimizer
     /// The momentum factor. It accelerates stochastic gradient descent in the relevant direction
     /// and dampens oscillations.
     public var momentum: Float
-    /// The weight decay.
+    /// The learning rate decay.
```

Comment:
In `Sources/TensorFlow/Optimizers/MomentumBased.swift`, there's another occurrence of `/// The weight decay.` that should be renamed to `/// The learning rate decay.`, if you'd like to update that as well!

Reply:
Changed this one as well.